app.js

```diff
@@ -2,10 +2,15 @@ const express = require('express')
 const  app = express()
 require('dotenv').config()
 require('express-async-errors')
-const session = require("express-session")
 
-// const connectDB = require('./db/connect')
+// routes 
+const itemRoutes = require('./routes/items')
+
+const session = require("express-session")
 
+const connectDB = require('./db/connect')
+const csrf = require('csurf')
```

Comment:
This isn't the package the assignment recommended, curious why you went with this one?

Reply:
firstly I used HOST-CSRF package as assignment recommended, but I got an error and couldn't solve it. 
Then I used CSURF package.